docs/UserGuide.md

```diff
@@ -289,6 +289,13 @@ Format: `find_person KEYWORD [MORE_KEYWORDS]`
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 * `find_person` searches the name of the `Person` and `Group` that they are assigned 
 to and will display them accordingly.
+* Note the following scenario:
```

Comment:
This segment might be better placed in a note or warning box. It might also be better to place it under `edit_person` as the user might try to refer to the `edit_person` segment when a person disappears after the `edit_person` command